src/lib/containers/row_renderers/ObservationCard.tsx

```diff
@@ -46,7 +45,10 @@ export const ObservationCard: React.FunctionComponent<ObservationCardProps> = ({
       {time && (
         <div className="ObservationCard__time">
           <IconSvg icon="time" />
-          <span>{getFriendlyPreciseDuration(time, timeUnits)}</span>
+          <span>
+            {`${time} ${timeUnits}`}
+            {Number(time) > 1 ? 's' : ''}
+          </span>
```

Comment:
Removing `getFriendltyPreciseDuration` and replacing with above being this is the only place used and not compatible with dayjs

Reply:
How is it not compatible with dayjs? It seems like the behavior here will change, but I can't tell for sure because the tests were removed. Please keep the function and tests. If the function can't be trivially re-implemented or the expected test values need to change, then let's work through that.

Reply:
After replacing momentjs with dayjs for getFriendlyPreciseDuration, one of the tests fail as the following.
```
    expect(received).toBe(expected) // Object.is equality

    Expected: "1 year 1 month"
    Received: "1 year 25 days"

      17 |     expect(getFriendlyPreciseDuration('4', 'week')).toBe('28 days')
      18 |     expect(getFriendlyPreciseDuration('6', 'week')).toBe('1 month 12 days')
    > 19 |     expect(getFriendlyPreciseDuration('13', 'month')).toBe('1 year 1 month')
         |                                                       ^
      20 |   })
      21 | })
```